src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -33,7 +35,7 @@
   <ItemGroup Condition="'$(Configuration)' == 'Debug' and '@(_MonoComponent->Count())' == 0">
     <_MonoComponent Include="hot_reload;debugger" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(MonoLightweightMarshallerIncompatibleAssemblies)' != ''">
```

Comment:
@maraf @radical This only affects user builds that use WasmBuildNative, right?

Where do we set up the defaults for simple projects that aren't doing WasmBuildNative - that don't have any pinvokes except for whatever corelib needs

Reply:
That would be in `WasmApp.targets`.

Reply:
Isn't this an item? `@(MonoLightweightMarshallerIncompatibleAssemblies -> Count()) > 0`